tests/integration/integration-test.py

```diff
@@ -1,5 +1,7 @@
+from datetime import datetime
```

Comment:
We usually put `from ... import ...` after `import ...` section and add a blank line between them. For example, https://github.com/osu-mist/notes-api/blob/master/tests/integration/integration-test.py#L1-L8